index.html

```diff
@@ -85,14 +85,7 @@ <h4 class="modal-title white-text w-100 font-weight-bold py-2">New Book</h4>
                 </div>
                 <div class="field">
                   <label class="label">Read Status</label>
-                  <div class="control">
-                    <div class="select">
-                      <select name="status" id="status">
-                        <option value="true">Read</option>
-                        <option value="false">Not read</option>
-                      </select>
-                    </div>
-                  </div>
```

Comment:
It seems like you remove this. Why?
I think it caused the error mentioned above.